lib/exception_handling.rb

```diff
@@ -387,7 +387,8 @@ def enhance_exception_data(data)
         ExceptionHandling.custom_data_hook.call(data)
       rescue Exception => ex
         # can't call log_error here or we will blow the call stack
-        log_info( "Unable to execute custom custom_data_hook callback. #{encode_utf8(ex.message)} #{ex.backtrace.each {|l| "#{l}\n"}}" )
+        traces = ex.backtrace.map{ |l| "#{l}\n" }.join
+        log_info( "Unable to execute custom custom_data_hook callback. #{encode_utf8(ex.message)} #{traces}" )
```

Comment:
FYI, logger.info is fine here, because it goes "directly" to the file.  Yeah, that could blow up, but it is pretty direct so we have to have faith at some point.

```
def log_info( message )
      ExceptionHandling.logger.info( message )
    end
```